llvm/include/llvm/Transforms/Utils/BuildLibCalls.h

```diff
@@ -251,7 +251,7 @@ namespace llvm {
 
   /// Emit a call to the calloc function.
   Value *emitCalloc(Value *Num, Value *Size, IRBuilderBase &B,
-                    const TargetLibraryInfo &TLI);
+                    const TargetLibraryInfo &TLI, unsigned AddrSpace = 0);
```

Comment:
To prevent mistakes, can we make this a required parameter?

Reply:
We can, I did not because it seems that most interfaces pass it as a defaulted parameter. I do agree with you that it'd be better not to do that though, and since this is only caller in one place, the impact is limited.